packages/slate-react/src/components/editable.tsx

```diff
@@ -513,7 +512,12 @@ export const Editable = (props: EditableProps) => {
           }
           data-slate-editor
           data-slate-node="value"
-          contentEditable={readOnly ? undefined : true}
+          // explicitly set this
+          contentEditable={!readOnly}
+          // in some cases, if use a decoration which range are relate with selection to decorate a text node,
```

Comment:
I'm not able to fully understand this comment... could you please clean up the grammar, and maybe provide a link to a resource or example that explains the use of zindex=-1?